src/sentry/issues/grouptype.py

```diff
@@ -144,5 +144,25 @@ class PerformanceUncompressedAssetsGroupType(GroupType):
 class ProfileBlockedThreadGroupType(GroupType):
     type_id = 2000
     slug = "profile_blocked_thread"
-    description = "Blocked Thread"
+    category = GroupCategory.PROFILE.value
+
+
+@dataclass(frozen=True)
+class ProfileFileIOGroupType(GroupType):
+    type_id = 2001
+    slug = "profile_file_io_main_thread"
+    category = GroupCategory.PROFILE.value
```

Comment:
Can you add a description field?

Reply:
Is description really useful when I'm passing the issue title in the occurrence? Do you except something different in the description?

Reply:
Answered by DM: it will be used in emails or Slack and could be different than issue title.